libs/sdk-ui-dashboard/src/presentation/topBar/menuButton/DefaultMenuButton.tsx

```diff
@@ -26,24 +28,151 @@ const bubbleAlignPoints: IAlignPoint[] = [{ align: "cl tr" }];
 export const DefaultMenuButton = (props: IMenuButtonProps): JSX.Element | null => {
     const { menuItems } = props;
     const [isOpen, setIsOpen] = useState(false);
+    const [autofocusSubmenu, setAutofocusSubmenu] = useState(false);
     const intl = useIntl();
     const tooltipText = intl.formatMessage({ id: "controlButtons.options.tooltip" });
+    const backLabel = intl.formatMessage({ id: "controlButtons.options.back" });
+    const closeLabel = intl.formatMessage({ id: "controlButtons.options.close" });
+    const menuWrapperRef = useRef<HTMLDivElement>(null!);
```

Comment:
Why is `!` necessary here? Possibly even the init value?

Reply:
fixed